src/dgcgears.c

```diff
@@ -1898,6 +1917,9 @@ main(int argc, char *argv[])
    if (!check_sample_count_support(sample_count))
       error("Sample count not supported");
 
+   if (!check_indirect_commands_graphics_support())
+      error("Indirect execution does not support graphics %s", use_shader_object ? "shader" : "pipeline");
```

Comment:
Also this should say `does not support graphics X switching`

Reply:
Done.

Reply:
Great, thanks